<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPatientPage</class>
 <widget class="QWidget" name="AddPatientPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Patient</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="first_name">
     <property name="placeholderText">
      <string>First Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="last_name">
     <property name="placeholderText">
      <string>Last Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="age">
     <property name="placeholderText">
      <string>Age</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="gender">
     <item>
      <property name="text">
       <string>Male</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Female</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="identification_card">
     <property name="placeholderText">
      <string>Identification Card</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="address">
     <property name="placeholderText">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="phone">
     <property name="placeholderText">
      <string>Phone</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateEdit" name="date">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="description">
     <property name="placeholderText">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="prescription">
     <property name="placeholderText">
      <string>Prescription</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="submit_btn">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
